ui-tests/src/test/java/org/openmrs/reference/page/HtmlFormsPage.java

```diff
@@ -0,0 +1,29 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class HtmlFormsPage extends Page {
+
+    private static final By NEW_HTML_FORM = By.cssSelector("a[href='htmlForm.form']");
+    private static final By NAME = By.name("form.name");
+    private static  By DESCRIPTION = By.name("form.description");
+    private static  By SAVE_FORM = By.cssSelector("input[type=\"submit\"]");
```

Comment:
avoid escaping which makes things not readable enough, replace those with single quotes across the PR